packages/app-js/src/snippets/storage-examples.ts

```diff
@@ -11,13 +11,14 @@ export const storageGetInfo: Snippet = {
   code: `// Get chain state information
 // Make our basic chain state/storage queries, all in one go
 
-const [minimumPeriod, validators, transferFee] = await Promise.all([
-  api.query.timestamp.minimumPeriod(),
-  api.query.session.validators(),
-  api.query.balances.transferFee()
+const [blockPeriod, validators] = await Promise.all([
+  api.query.timestamp.blockPeriod(),
```

Comment:
Sorry, dang tough because we can't visually inspect an actual newly-compiled node, in this case I'm guessing you looked at a week-old metadata,. updated here - https://github.com/polkadot-js/api/pull/1142/files#diff-a60854aee74a1ede6dc8edf6544d4240R349

Reply:
Yeah, I thought docker-compose would pull the latest versions of the containers automatically, I just found out that you need to do that manually.